### PR TITLE
Update sh_voices.lua

### PR DESCRIPTION
**Pull request by Truba55**
Deleted several "Schema.voices.Add" which did not work. (In German as well as in English)
In line 263 there is no WAV extension. With .wav it still does not work.
In line 343 I corrected one of the punctuation marks
Line 299 is self-explanatory

### CODE REVIEW

File: schema/sh_voices.lua [hunk 1]
```diff
@@ -296,7 +294,7 @@ Schema.voices.Add("Combine", "SEARCH", "Search!", "npc/metropolice/vo/search.wav
 Schema.voices.Add("Combine", "SEARCHING FOR SUSPECT", "Searching for suspect.", "npc/metropolice/vo/searchingforsuspect.wav")
 Schema.voices.Add("Combine", "SECOND WARNING", "This is your second warning!", "npc/metropolice/vo/thisisyoursecondwarning.wav")
 Schema.voices.Add("Combine", "SECTOR NOT STERILE", "Confirmed- sector not sterile.", "npc/combine_soldier/vo/confirmsectornotsterile.wav")
-Schema.voices.Add("Combine", "SECTOR SECURE", "Sector is now secure.", "npc/combine_soldier/vo/sectorisnotsecure.wav")
+Schema.voices.Add("Combine", "SECTOR NOT SECURE", "Sector is not secure.", "npc/combine_soldier/vo/sectorisnotsecure.wav")
```
Comment (Truba55) [inline schema/sh_voices.lua:299]: I've listened to it ingame and it's supposed to be "not". 
Reasons for this:
npc/combine_soldier/vo/sectoris**not**secure.wav
and
in German it is also "not"

File: schema/sh_voices.lua [hunk 2]
```diff
@@ -144,7 +144,6 @@ Schema.voices.Add("Combine", "GO SHARP", "Go sharp!", "npc/combine_soldier/vo/go
 Schema.voices.Add("Combine", "GOING IN", "Cover me, I'm going in!", "npc/metropolice/vo/covermegoingin.wav")
 Schema.voices.Add("Combine", "GOT A DB", "Uh, we got a DB here, cancel that 11-42.", "npc/metropolice/vo/wegotadbherecancel10-102.wav")
 Schema.voices.Add("Combine", "GOT HIM AGAIN", "Got him again, suspect is 10-20 at...", "npc/metropolice/vo/gothimagainsuspect10-20at.wav")
-Schema.voices.Add("Combine", "GOT ONE ACCOMPLICE", "I got one accomplice here!", "npc/metropolice/vo/gotoneaccompliceherea.wav")
```
Comment (impulsh) [inline schema/sh_voices.lua:147]: This should be changed to `npc/metropolice/vo/gotoneaccomplicehere.wav`

File: schema/sh_voices.lua [hunk 3]
```diff
@@ -260,7 +259,6 @@ Schema.voices.Add("Combine", "PRIORITY 2", "I have a priority-two anti-citizen h
 Schema.voices.Add("Combine", "PRODUCTION BLOCK", "Production-block.", "npc/metropolice/vo/productionblock.wav")
 Schema.voices.Add("Combine", "PROSECUTE MALCOMPLIANT", "Ready to prosecute malcompliant citizen, final warning issued!", "npc/metropolice/vo/readytoprosecutefinalwarning.wav")
 Schema.voices.Add("Combine", "PROSECUTE", "Prosecute!", "npc/metropolice/vo/prosecute.wav")
-Schema.voices.Add("Combine", "PROSECUTING", "Prosecuting.", "npc/combine_soldier/vo/procecuting.")
```
Comment (impulsh) [inline schema/sh_voices.lua:263]: This should be changed to `npc/combine_soldier/vo/prosecuting.wav`

File: schema/sh_voices.lua [hunk 4]
```diff
@@ -316,15 +314,13 @@ Schema.voices.Add("Combine", "STAR", "Star.", "npc/combine_soldier/vo/star.wav")
 Schema.voices.Add("Combine", "STATIONBLOCK", "Stationblock.", "npc/metropolice/vo/stationblock.wav")
 Schema.voices.Add("Combine", "STAY ALERT", "Stay alert.", "npc/combine_soldier/vo/stayalert.wav")
 Schema.voices.Add("Combine", "STERILIZE", "Sterilize!", "npc/metropolice/vo/sterilize.wav")
-Schema.voices.Add("Combine", "STINGER", "Stinger.", "npc/combine_soldier/vo/stringer.wav")
```
Comment (impulsh) [inline schema/sh_voices.lua:319]: This should be changed to `npc/combine_soldier/vo/stinger.wav`

File: schema/sh_voices.lua [hunk 5]
```diff
@@ -316,15 +314,13 @@ Schema.voices.Add("Combine", "STAR", "Star.", "npc/combine_soldier/vo/star.wav")
...
 Schema.voices.Add("Combine", "STORM", "Storm.", "npc/combine_soldier/vo/storm.wav")
 Schema.voices.Add("Combine", "STRIKE", "Striker.", "npc/combine_soldier/vo/striker.wav")
 Schema.voices.Add("Combine", "SUBJECT 505", "Subject is 505!", "npc/metropolice/vo/subjectis505.wav")
 Schema.voices.Add("Combine", "SUBJECT HIGH SPEED", "All units, be advised, subject is now high-speed!", "npc/metropolice/vo/subjectisnowhighspeed.wav")
 Schema.voices.Add("Combine", "SUBJECT", "Subject!", "npc/metropolice/vo/subject.wav")
 Schema.voices.Add("Combine", "SUNDOWN", "Sundown.", "npc/combine_soldier/vo/sundown.wav")
 Schema.voices.Add("Combine", "SUSPECT INCURSION", "Disp reports suspect-incursion at location.", "npc/metropolice/vo/dispreportssuspectincursion.wav")
-Schema.voices.Add("Combine", "SUSPECT MOVED TO", "Suspect has moved now to...", "npc/metropolice/vo/supsecthasnowmovedto.wav")
```
Comment (impulsh) [inline schema/sh_voices.lua:327]: This should be changed to `npc/metropolice/vo/supsecthasmovednowto.wav` (typo included)

File: schema/sh_voices.lua [hunk 6]
```diff
@@ -361,17 +357,14 @@ Schema.voices.Add("Combine", "VERDICT", "You want a malcompliance verdict?", "np
 Schema.voices.Add("Combine", "VICE", "Vice!", "npc/metropolice/vo/vice.wav")
 Schema.voices.Add("Combine", "VICTOR", "Victor!", "npc/metropolice/vo/victor.wav")
 Schema.voices.Add("Combine", "VISCON", "Viscon.", "npc/combine_soldier/vo/viscon.wav")
-Schema.voices.Add("Combine", "VISUAL EXOGEN", "Visual on exogen.", "npc/combine_soldier/vo/visualonexogen.wav")
```
Comment (impulsh) [inline schema/sh_voices.lua:364]: This should be changed to `npc/combine_soldier/vo/visualonexogens.wav`

**Review by impulsh — CHANGES_REQUESTED** [review]

**Pull request merged**